packages/backend/src/configuration/environment.ts

```diff
@@ -34,47 +40,48 @@ const envVarsSchema = object({
   MOODLE_BASE_URL: string()
     .required()
     .uri()
-    .description('URL of your Moodle instance, eg. https://moodle.domain.me'),
+    .description('URL of your Moodle instance, eg. https://moodle.domain.me .'),
   MOODLE_FETCH_INTERVAL: number()
     .default(900000)
-    .description('Interval to look for updates on moodle (in ms)'),
+    .description('Interval to look for updates on moodle (in ms).'),
```

Comment:
```suggestion
    .description('Interval to look for updates on Moodle (in ms).'),
```
Only for uniform wording.